### PR TITLE
Phase 2: Notifications sent on Events

### PR DESCRIPTION
**Pull request by devinmatte**

### CODE REVIEW

File: config.env.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 DEBUG = False
 IP = environ.get("PACKET_IP", "localhost")
 PORT = environ.get("PACKET_PORT", "8000")
+PROTOCOL = environ.get("PACKET_PROTOCOL", "https://")
```
Comment (JoelEager) [inline config.env.py]: Why does this need to be a thing? Couldn't we just always assume `https`? Not questioning it; just curious.

File: packet/notifications.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+        notification = onesignal.Notification(post_body=post_body)
+        notification.post_body["content"]["en"] = signer + ' signed your packet! Congrats or I\'m Sorry'
+        notification.post_body["headings"]["en"] = 'New Packet Signature!'
+        notification.post_body["chrome_web_icon"] = 'https://profiles.csh.rit.edu/image/' + signer
+        notification.post_body["include_player_ids"] = tokens
+
+        onesignal_response = onesignal_client.send_notification(notification)
+        print(onesignal_response.status_code)
+        print(onesignal_response.json())
```
Comment (JoelEager) [inline packet/notifications.py]: Why are there print statements here? This doesn't seem to be called by the CLI.

File: packet/notifications.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        notification = onesignal.Notification(post_body=post_body)
+        notification.post_body["content"]["en"] = packet.freshman.name + ' got 💯 on packet!'
+        notification.post_body["headings"]["en"] = 'New 100% on Packet!'
+        # TODO: Issue #156
+        notification.post_body["chrome_web_icon"] = 'https://profiles.csh.rit.edu/image/' + packet.freshman_username
+        notification.post_body["include_player_ids"] = tokens
+
+        onesignal_response = onesignal_client.send_notification(notification)
+        print(onesignal_response.status_code)
+        print(onesignal_response.json())
```
Comment (JoelEager) [inline packet/notifications.py]: Same here

File: config.env.py (hunk 1 above)
Comment (devinmatte) [reply config.env.py]: When testing locally we need to use http, so it will almost always be https, but can't be guaranteed

**Pull request merged**